appveyor.yml

```diff
@@ -14,10 +14,7 @@ build_script:
   - cmd: echo vs %VersionSuffix%"
   - cmd: build.bat
 test_script:
-  - cmd: dotnet run --project .\test\Tests.SystemJson\Tests.SystemJson.fsproj -c Release -f net461
-  - cmd: dotnet run --project .\test\Tests.FSharpData\Tests.FSharpData.fsproj -c Release -f net5
```

Comment:
better not forget this

Reply:
Don't worry, the plan is to revert these CI files when I have all 4 libs working.